autogen/docs/behaviorspace.md.mustache

```diff
@@ -109,7 +109,9 @@ or view sliders in the Interface Tab (*Since 6.4*).
 
 The information that can be included in the Experiment dialog is detailed below.
 Note that it is not necessary to specify everything; some parts can be left blank
-or with their default values.
+or with their default values. If a property's name is underlined, you can hover over
+it to read more info about the property and how it is used, so you don't have to
+revisit this page as often.
```

Comment:
Please add *(Since 6.4)* and correct the misplaced period below.

revisit this page as often *(Since 6.4)*.

**Experiment name:** Experiments in the same model must have different names. If
you open a model that contains experiments with duplicate names, the conflicting
names will be altered to ensure that all experiment names remain unique *(Since 6.4)*.